src/index.js

```diff
@@ -1 +1,32 @@
-export default () => true
+import defaultsDeep from 'lodash.defaultsdeep';
+import pickBy from 'lodash.pickby';
+
+class Stipulate {
+
+  constructor(options) {
+    this.baseOptions = options;
+  }
+
+  prefix(request) {
+    return request;
+  }
+
+  postfix(response) {
```

Comment:
I agree that the name isn't particularly clear, but `afterSend` doesn't seem like an improvement. I get that it's saying "this happens after the request has been made" but what does that mean in terms of the response? Is the response involved? or is this just a hook to be executed once you've sent the request?

alternatives I've thought about:
- `onResponse`
- `whenReturned`
- `onceReceived`

I'm not particularly fond of any of those though